### DISCUSSION TITLE
Does Quarto support "unfurling" in clients like Slack?

### DISCUSSION THREAD

**Discussion by mfisher87** [quarto-dev/quarto-cli#9023] [Q&A] [answered]
### Description

When I share links in Slack, sometimes they do this "unfurl" thing:

![image](https://github.com/quarto-dev/quarto-cli/assets/3608264/b4f0b1fc-06e3-442b-aa55-61d6af263f4b)

But when I post links to my Quarto sites, they don't do that (1.3 and 1.4). Is there current or planned support for this feature? Perhaps I'm using the wrong terminology when I'm searching the docs?

Thanks all for your time!

**Comment by jimjam-slam** [comment, marked as answer]
They sure do! Here's a Quarto link I just posted in my work Slack on Friday (a self-serving plug — it's a chart we just made 😉):

![image](https://github.com/quarto-dev/quarto-cli/assets/6520659/c2301793-a18e-4efd-a0ac-ea3c65623aea)

You'll want to fill in the [social metadata](https://quarto.org/docs/websites/website-tools.html#social-metadata) on a document (I believe Slack specifically uses the OpenGraph ones, but I'm not 100% sure).

If you need an example of how to do it, [here's the source for the above example](https://github.com/360-info/report-women-unpaid-work/blob/a398b2fe828b19e723aaf73262f6f93c041335b9/timeuse-meta/index.qmd#L6C1-L14C31). Things to note are:

- Use `pagetitle` if you don't already have a `title`, and use `description` for the longer text below the title
- Define an `image` under `open-graph` to get a thumbnail. You may also need to define that image path as a `resource` on either the document itself or your `_quarto.yml`, depending on whether the path to it is relative or from the project root

**Reply by jimjam-slam** [reply]
Also, [opengraph.xyz](https://www.opengraph.xyz/) is a useful site for verifying whether you've configured it correctly!

**Reply by mfisher87** [reply]
@jimjam-slam Thanks for the answer! It's really useful to know the name of the protocol and have a testing tool, because this has sometimes worked for me in the past, and other times not.

:bow:

**Reply by mcanouil** [reply]
Note that these kind of changes are not instantaneous as there are many caching systems.
Depending on the website/tool, it can take up to 48 hours to propagate.

For instance on LinkedIn, there is a "trick" to flush the LinkedIn cache which is to paste your URL into <https://www.linkedin.com/post-inspector/>.

**Reply by mfisher87** [reply]
>Note that these kind of changes are not instantaneous as there are many caching systems. Depending on the website/tool, it can take up to 48 hours do propagate.

Ahhh, that's really useful to know as well! Thank you!
